spec/features/adding_items_to_cart_spec.rb

```diff
@@ -155,6 +155,130 @@
         end
       end
     end
+
+    context "when both of the bundle items have a user-selectable variant", js: true do
+      let(:bundle) { create(:product_in_stock, name: "Bundle", sku: "BUNDLE") }
+      let(:keychain) do
+        create(:product_in_stock, name: "Keychain",
+                                  sku: "KEYCHAIN",
+                                  can_be_part: true)
+      end
+      let(:shirt) do
+        shirt, _shirts_by_size = create_bundle_product_with_options(
+          name: "Shirt",
+          sku: "SHIRT",
+          option_type: "Size",
+          option_values: ["Large", "XL"]
+        )
+
+        shirt
```

Comment:
Does it make sense to create another helper method that calls `create_bundle_product_with_options` internally but return only the product?